modules/ROOT/pages/spring/spring-module.adoc

```diff
@@ -81,9 +77,32 @@ Note that the Spring configuration must be valid. Here is an example:
 </beans>
 ----
 
-IMPORTANT: The beans files used with the Spring Module must be exported, if not the application won't start. See xref:4.1@mule-runtime::how-to-export-resources.adoc[How to Export Resources].
+The beans files used with the Spring module must be exported, if not the application won't start. See xref:4.1@mule-runtime::how-to-export-resources.adoc[How to Export Resources]. 
+
+If you are using Anypoint Studio 7.2 or later, you don't need to manually update the `mule-artifact.json` file. 
+
+Mule still needs a descriptor where the Spring beans are declared, so the embedded packager within Studio auto generates the descriptor with all the resources it picks up from the project. However, if you choose to declare the resources you need to export, then Studio packages only those resources.
```

Comment:
```suggestion
Because Mule also needs a descriptor where the Spring beans are declared, the embedded packager within Studio automatically generates one with all the resources it picks up from the project. However, if you choose to declare the resources you need to export, then Studio packages only those resources.
```